app/src/main/java/app/revanced/manager/ui/component/patches/OptionFields.kt

```diff
@@ -354,7 +376,9 @@ private object BooleanOptionEditor : OptionEditor<Boolean> {
 
     @Composable
     override fun ListItemTrailingContent(scope: OptionEditorScope<Boolean>) {
-        HapticSwitch(checked = scope.current, onCheckedChange = scope.setValue)
+        HapticSwitch(checked = scope.current, onCheckedChange = {
+            scope.showSelectionWarningDialogElse { scope.setValue }
+        })
```

Comment:
Can you add newlines here to make the formatting more consistent with the rest of the project?

Reply:
Also, I think `onCheckedChange` is broken now since it is not calling `scope.setValue`.